kex-runner/src/main/kotlin/org/vorpal/research/kex/asm/analysis/symbolic/InstructionSymbolicChecker.kt

```diff
@@ -17,10 +17,22 @@ class InstructionSymbolicChecker(
     ctx: ExecutionContext,
     rootMethod: Method,
 ) : SymbolicTraverser(ctx, rootMethod) {
-    override val pathSelector: SymbolicPathSelector = DequePathSelector()
+    override val pathSelector: SymbolicPathSelector
     override val callResolver: SymbolicCallResolver = DefaultCallResolver(ctx)
     override val invokeDynamicResolver: SymbolicInvokeDynamicResolver = DefaultCallResolver(ctx)
 
+    init {
+        val pathSelectorName: String = kexConfig.getStringValue("kex", "pathSelector", "NSubpathPathSelector")
```

Comment:
There is a special section in config file dedicated to symbolic mode, accordingly it is called "symbolic". Everything connected to symbolic configuration should to be specified there.